defaults-jalien.sh

```diff
@@ -6,45 +6,16 @@ env:
   CXXFLAGS: "-fPIC -g -O2 -std=c++11"
   CFLAGS: "-fPIC -g -O2"
   CMAKE_BUILD_TYPE: "RELWITHDEBINFO"
+  USE_AUTORECONF: "false"
```

Comment:
Is USE_AUTORECONF actually used? IIRC env does not apply for current package, but only for the dependent ones.

Reply:
It looks like it works, it's defined in a defaults file and seems to be visible for all packages.

```
DEBUG:autotools:autotools:v1.5.0: + source /alice/sw/SPECS/slc7_x86-64/autotools/v1.5.0-1/autotools.sh
...
DEBUG:autotools:autotools:v1.5.0: ++ USE_AUTORECONF=false
DEBUG:autotools:autotools:v1.5.0: ++ echo 'Building ALICE autotools. To avoid this install autoconf, automake, autopoint, texinfo, pkg-config.'
```